src/config.js

```diff
@@ -182,6 +189,9 @@ Config.prototype.readProjectFile = function(p, options) {
     if (options.fallback) {
       return options.fallback;
     }
+    if (e.code === "ENOENT") {
+      throw new FirebaseError(`Rules file not found: ${this.path(p)}`, { original: e });
```

Comment:
This should just be "File not found" since `readProjectFile` is not necessarily only for rules.